docs/develop/08_opendid/04_integrate_opendid.md

```diff
@@ -0,0 +1,117 @@
+---
+id: integrate_opendid
+title: Integrate OpenDID
+---
+
+OpenDID follows the [OpenID Connect 1.0 Specification](https://openid.net/specs/openid-connect-core-1_0.html#Introduction) and implements both the [implicit flow](https://openid.net/specs/openid-connect-core-1_0.html#ImplicitFlowSteps)
+and the [authorization code flow](https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth).
+Read the [demo project guide](05_demo_project.md) for an example of integrating OpenDID.
+
+## Authorization code flow
+
+Initiate the flow by redirecting to the **GET** `/api/v1/authorize` endpoint on the OpenDID service and setting the following query
+parameters:
```

Comment:
I guess they have to be URL-encoded, right? Is that implicit with the fact that they are query parameters?